modules/ossm-cr-example.adoc

```diff
@@ -67,9 +67,10 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
         traceSampling: 100.0
 
       kiali:
-       dashboard:
-          user: admin
-          passphrase: admin
+        enabled: true
+
       tracing:
         enabled: true
+        jaeger:
+          template: all-in-one
```

Comment:
When installing through the openshift console we should also point out that customers should switch the template from the default, production-elasticsearch, to the above all-in-one variant for all but large scale production use cases.

Reply:
The above will no longer be necessary, we've decided to update the metadata and republish the operator image using all-in-one as the defgault